### DISCUSSION TITLE
NAMD configuration files not found in sirah_x2.3_24-07.amber

### DISCUSSION THREAD

**Discussion by pedrofischerbioinfo** [SIRAHFF/documentation#80] [Q&A] [answered]
Hello, im Pedro Fischer, and im following the sirah tutorial for NAMD.
On the sirah_x2.3_24-07.amber file that o downloaded on 30/09/2024, i couldn't find the directory /tutorial/NAMD/

**Comment by SIRAHFF** [comment, marked as answer]
We just put up a new sirah_x2.3_24-07.amber.tar.gz file with a folder for NAMD tutorial.
You will need to redownload and unzip it in order to obtain this folder.

**Reply by pedrofischerbioinfo** [reply]
Thank you for the new version containing the NAMD configurations files.
I noticed that in the heat.conf file the positional restraint block is not activated "0": 

#Constraints on the Backbone 
if {0} {                                             ;# If 1 read the block 
constraints          on                        ;# Turns on constraints 
consref              ../protein_restraints.pdb ;# Reference PDB file for constraint positions 
conskfile            ../protein_restraints.pdb ;# File containing constraint force constants 
constraintScaling    1                         ;# Scaling factor for constraint forces 
consexp              2                         ;# Exponent for constraint potential 
conskcol             B                         ;# Column for constraint forces in the reference file (B for force constants) 
}
